plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/transport/kafka/OpenLineageKafkaTransport.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.openlineage.transport.kafka;
+
+import com.google.inject.Inject;
+import io.openlineage.client.transports.KafkaConfig;
+import io.openlineage.client.transports.KafkaTransport;
+import io.openlineage.client.transports.Transport;
+import io.trino.plugin.openlineage.config.kafka.OpenLineageClientKafkaTransportConfig;
+import io.trino.plugin.openlineage.transport.OpenLineageTransport;
+
+import java.util.Properties;
+
+public class OpenLineageKafkaTransport
```

Comment:
Could we add test like `TestOpenLineageEventListenerMarquezIntegration.java` but with Kafka? 

See also https://github.com/trinodb/trino/pull/22888 for similar test with Kafka cc @marton-bod